perfkitbenchmarker/azure/__init__.py

```diff
@@ -11,3 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from perfkitbenchmarker.azure import util
+
+util.CheckAzureVersion()
```

Comment:
I don't think we should warn when the cloud isn't Azure. Perhaps you could subscribe to [`initialization_complete`](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/events.py#L29) and warn conditionally?